python/determineLuminosity.py

```diff
@@ -227,6 +228,9 @@ def simulateDataOnHimster(
                     sim_par = thisExperiment.simParams
                     #! these mus be applied again, because they change at run time
                     # (the config on disk doesn't change and doesn't know this)
+                    sim_par.sim_type=SimulationType.BOX
+                    sim_par.num_events_per_sample=box_num_events_per_sample
+                    sim_par.num_samples=box_num_samples
```

Comment:
No, line 227 sim_par = `thisExperiment.simParams` just set the dpm gen.

Reply:
Oh wait, those are two different things. Yes you're right, for the comparison data we need box_num_* :+1: 

But we cannot hard-code `sim_par.sim_type=SimulationType.BOX` like that (even though it was like that before) because KOALA needs the DPM generator for the comparison data because of the needed recoil partner. BOX gen cannot supply that. So we cannot hardcode, but we can add the option to the recoParams in the config and the config generator.

However we shouldn't impoort simulation.py into reconstruction.py, but instead both should import `SimulationType.py`, which should contain only that enum.

Reply:
Sure I will do that.